src/dev_bot.py

```diff
@@ -8,24 +8,34 @@
 
 import logging
 
-from aiogram import types
+from aiogram import types, Bot
 
 from bot_behaviors.random_responses import random_response
 from bot_behaviors.experience import Experience
 from database.database import Database
-
+from decouple import config
+bot = Bot(token= config('API_TOKEN'))
 
 class DevBot:
     def __init__(self, dispatcher):
         self.dispatcher = dispatcher
+        self.bot = bot
         self.database = Database()
         self.experience = Experience(self.database)
         self.run_bot()
 
     def run_bot(self):
+        async def expname(chatid, userid):
+            member = await bot.get_chat_member(chat_id=str(chatid), user_id=str(userid))
+            return member['user']['first_name']
+            
         @self.dispatcher.message_handler(commands=['exp'])
         async def exp(message: types.Message):
-            response = self.database.get_experiences(chat_id=message.chat.id)
+            experiences_db = self.database.get_experiences(chat_id=message.chat.id)
+            response = "Experiências:\n\n"
+            for experience in experiences_db:
+                re = await expname(experience[1], experience[0])
```

Comment:
Talvez tenhamos problema de performance aqui, se nós chamarmos o "/exp 30" ou seja, o top 30 de xp, ele vai fazer 30 requisições pra API do Telegram, talvez isso tenha uma demora notável, ou não. Precisamos testar isso antes do merge deste PR. Vamos criar um bot de teste, subir no servidor atual do Devbot, e testar ele no grupo Developers.

Reply:
está sendo feito de forma assíncrona, não haverá problemas